### PR TITLE
Fix inline filtered elements

### PR DESCRIPTION
**Pull request by Adenilson**
It will create a StackingContext for inline elements that may need it, I'm looking for feedback on this. 

Not for commit, as there are some tests failing.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/5782)

### CODE REVIEW

File: components/layout/display_list_builder.rs [hunk 1]
```diff
@@ -1624,3 +1574,49 @@ impl ToGfxColor for RGBA {
         color::rgba(self.red, self.green, self.blue, self.alpha)
     }
 }
+
+fn create_stacking_context(fragment: &Fragment,
```
Comment (pcwalton) [inline components/layout/display_list_builder.rs]: Can you move this to be a method on `FragmentDisplayListBuilding`?

**Comment by pcwalton** [comment]
Yeah, this looks like the right approach.

**Comment by Adenilson** [comment]
Related to #5703.

**Comment by pcwalton** [comment]
One more comment and this should be good to go.

**Comment by Adenilson** [comment]
Done.

**Comment by pcwalton** [comment]
Looks good, squash away

**Comment by Adenilson** [comment]
Squashed.

**Pull request merged**